### ISSUE TITLE
Link to introductory blog post is broken

### ISSUE THREAD

**Issue by jeverling** [holoviz/geoviews#478]
Hi, the link to the blog post mentioned in the README 404s: https://www.continuum.io/blog/developer-blog/introducing-geoviews

I couldn't find out if the post exists in a new location.

**Comment by jbednar** [comment]
I can't find it either; the closest I see online is http://pythonic.zoomquiet.top/data/20160909102244/index.html . 

I don't know if this was the final version, but here's a version of it that was on my hard drive:
[geoviews_blog.zip](https://github.com/holoviz/geoviews/files/5503572/geoviews_blog.zip)

**Comment by ceball** [comment]
https://web.archive.org/web/20170713032855/https://www.continuum.io/blog/developer-blog/introducing-geoviews

Looks like it moved to anaconda then was later silently deleted.

At least the package itself was not also renamed 😂

**Comment by jbednar** [comment]
Yes, our website team has verified it was lost when they replaced the website with a new one, but they promise to reinstate it if we provide the source material...

**Comment by philippjfr** [comment]
Probably better to write an updated version since that's significantly out of date.
